extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/queries/QueryChange.java

```diff
@@ -37,9 +37,11 @@
 @DefaultAnnotation(NonNull.class)
 public final class QueryChange implements Serializable {
     private static final long serialVersionUID = 1L;
+
     private final UUID queryId;
     private final ChangeType changeType;
     private final Optional<String> sparql;
+    private final Optional<Boolean> isActive;
```

Comment:
Why can't this just be a primitive boolean so a null value isn't possible?

Reply:
Null objects are ambiguous and prone to NPEs, so I avoid using them.